test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -434,6 +434,26 @@ export function setup(opts: minimist.ParsedArgs) {
 				await app.workbench.sqlNotebook.textCellToolbar.changeTextCellView('Markdown View');
 				await app.workbench.sqlNotebook.waitForActiveCellEditorContents(s => s.includes('- **_<u><mark>Markdown Test</mark></u>_**'));
 			});
+
+			it('can save WYSIWYG notebook', async function () {
+				const app = this.app as Application;
+				const filename = 'emptyNotebook.ipynb';
+				await app.workbench.sqlNotebook.openFile(filename);
+
+				await app.workbench.sqlNotebook.addCell('markdown');
```

Comment:
It would be great to add comments with each "step" that's being done - while the function names are generally fairly clear having a comment explaining the overall "goal" of each step is useful for figuring out exactly what this is testing. 

(this applies to all smoke tests since those are generally made up of steps which consist of multiple discrete actions such as writing some text in WYSIWYG view and then bolding it)